modules/rosa-configure-privatelink-requirements.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/rosa-aws-privatelink.adoc
+
+[id="rosa-configure-privatelink-requirements.adoc_{context}"]
+= Requirements for using PrivateLink clusters
+
+For PrivateLink clusters, Internet gateways, NAT gateways and public subnets are not required, but the private subnets must have Internet connectivity provided in order to install required components. At least one single private subnet is required for Single-AZ clusters and at least 3 private subnets are required for Multi-AZ clusters. The following table shows the AWS resources that are required for a successful installation:
```

Comment:
Again, s/Internet connectivity/internet connectivity

when internet is not a multi-word term.

s/Multi A-Z/Multi-AZ

s/Single A-Z/Single-AZ

to match the rest of the ROSA docs.

Maybe make "The following table..." it's own paragraph to introduce the table? Because, in this case, it seems like a new thought from the first paragraph.

Reply:
@jonquilwilliams  Can you clarify?  I'm not finding instances of Multi A-Z or Single A-Z anywhere in the doc. Both Multi-AZ and Single-AZ seemed to be AWS specific. They have it capitalized (ex: Multi-AZ) in their doc except when used as an argument.

?
s/Multi A-Z/Multi-AZ
s/Single A-Z/Single-AZ

Reply:
My concussion is still impacting my eyesight a bit. I don;t see, not does a search reveal, a hyphen between A and Z (e.g. Multi A-Z). The correct abbreviation is Multi-AZ and Single-AZ, as I suggested. I'm not disputing that.

However, elsewhere in the documentation, you have "multi Availability Zone" and "mulitple Availabillity Zone", both of which should be changed to "multiple availability zone". So in other words, spell out "multiple" and use lower case for "availability" and "zone".